### ISSUE TITLE
Add Max CPU and Memory usage

### ISSUE THREAD

**Issue by ArvinTaghizadeh** [the-benchmarker/web-frameworks#5727]
Hi, please add max cpu and memory usage for each framework .

**Comment by waghanza** [comment]
Hi @ArvinTaghizadeh,

This is good idea ( in fact it is planned).

Since this project is not yet stable, this is probably a bit early to do this.

Regards,

**Comment by Joannis** [comment]
@waghanza are you open to revisiting this?

**Comment by waghanza** [comment]
yes, I have no ETA on this, it depends mostly on my personal schedules

I whish I had time, feel free to participate, would be great to :
- collect those metrics
- create a new UI (or update one) to expose them

**Comment by Joannis** [comment]
No worries, I'll happily take a stab at a PR - depending how airplane wifi treats me I might have something soon
